src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9407 @@
+    m_actionTemplates[ScriptAction::TEAM_MOVE_TOWARDS_NEAREST_OBJECT_TYPE].m_uiStrings[2] = " within ";
+
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_internalName =
+        "SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE";
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_uiName =
+        "Skirmish_/ Team attacks nearest group matching value comparison.";
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_numParameters = 3;
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_parameters[0] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_parameters[1] = Parameter::COMPARISON;
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_parameters[2] = Parameter::INT;
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_uiStrings[1] =
+        " attacks nearest group worth ";
+    m_actionTemplates[ScriptAction::SKIRMISH_ATTACK_NEAREST_GROUP_WITH_VALUE].m_uiStrings[2] = " ";
+
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_internalName =
+        "SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT";
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiName =
+        "Skirmish_/ Team performs command ability on most valuable object.";
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_numParameters = 4;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_parameters[0] =
+        Parameter::TEAM;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_parameters[1] =
+        Parameter::UNK_ABILITY_RELATED;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_parameters[2] =
+        Parameter::REAL;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_parameters[3] =
+        Parameter::BOOLEAN;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_numUiStrings = 5;
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiStrings[1] = " performs ";
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiStrings[2] =
+        " on most expensive object within ";
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiStrings[3] = " ";
```

Comment:
Is text missing here? Sentence makes no sense.

Reply:
This is correct.